{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Test defects","children":[{"name":"pydantic_core._pydantic_core.ValidationError: 1 validation error for UserSchema\ndata\n  Field required [type=missing, input_value={'success': False, 'statu...en 4 and 30 characters'}, input_type=dict]\n    For further information visit https://errors.pydantic.dev/2.11/v/missing","children":[{"name":"Изменение данных юзера","uid":"a577af58d5667d3","parentUid":"7a12ba45899ff1cc86fe1070ecaa4b22","status":"broken","time":{"start":1759918452556,"stop":1759918452636,"duration":80},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"7a12ba45899ff1cc86fe1070ecaa4b22"}],"uid":"bdbf199525818fae7a8651db9eafe741"}],"name":"categories"}